src/Payments.sol

```diff
@@ -276,23 +276,14 @@ contract Payments is
         address to,
         uint256 amount
     ) internal {
-        require(token != address(0), "token address cannot be zero");
-        require(to != address(0), "recipient address cannot be zero");
-
-        Account storage acct = accounts[token][msg.sender];
-
-        uint256 settleEpoch = settleAccountLockup(acct);
```

Comment:
I think we still need to settle here.

Reply:
We take care of that using a modifier in https://github.com/FilOzone/payments/pull/33.

But have reverted this change so we can merge this in main.

Reply:
I know, but each PR needs to stand on it's own. Otherwise I can't review them.